.github/workflows/operator_ci.yaml

```diff
@@ -152,22 +152,22 @@ jobs:
           version: v0.11.1
       - name: Wait for k8s cluster to be ready
         run: |
-          build/ci/wait_for_k8s_ready.sh
+          build/ci/github_actions/driver/wait_for_k8s_ready.sh
```

Comment:
Why is it under "driver" dir (instead of "operator")?

Reply:
I thought this is a driver-related dir, otherwise why would we need an `operator` dir? isn't it implied in the operator repo?

Reply:
and if this is a driver dir, does `wait_for_k8s_ready.sh` belong in it?

Reply:
@zingero ?

Reply:
I agree, we don't need an operator dir inside this repo.
I'm not sure where the `wait_fir_k8s_ready.sh` should be.

Reply:
IMO it can be directly in `github_actions`

Reply:
alright

Reply:
@oriyarde @zingero so where will be the other scripts? under `driver` dir? or under `github_actions` like `wait_for_k8s_ready.sh`?

Reply:
IMO the driver-related scripts should be under the `driver` dir

Reply:
done

Reply:
LGTM
@zingero is there anything else? if not, please resolve